create.yml

```diff
@@ -38,6 +34,14 @@
     - name: Add VM's to Tower and Database group
       include_role:
         name: add_hosts
+      when: install_tower | default('True') | bool
+
+    - name: Print IBM Cloud Instance Floating IPs
+      debug:
+        msg: 
+          - "IC instance Floating IP: "
+          - "{{ tower_node + db_node }}"
+      when: not install_tower | default('True') | bool 
```

Comment:
we should print IP's even when we are installing Tower. It makes it easy to find out which is the IP for Tower node and which is the IP for DB node.

Reply:
sure thing. thanks